src/utils/flatten.js

```diff
@@ -5,13 +5,18 @@ import isPlainObject from 'is-plain-object'
 import type { Interpolation } from '../types'
 
 export const objToCss = (obj: Object, prevKey?: string): string => {
-  const css = Object.keys(obj).map(key => {
-    if (isPlainObject(obj[key])) return objToCss(obj[key], key)
-    return `${hyphenate(key)}: ${obj[key]};`
-  }).join(' ')
-  return prevKey ? `${prevKey} {
+  const css = Object.keys(obj)
+    .filter(key => obj[key] === 0 || obj[key])
```

Comment:
Can you use the same check as the one we’re using inside flatten? It’d be nice to share it even if it shouldn’t make a difference here :)

Reply:
@wmertens Bump :wink: